docs/Guide/application-commands/application-command-registry/registering-chat-input-commands.mdx

```diff
@@ -3,56 +3,174 @@ sidebar_position: 3
+to change the name of the command. Normally if you would change the name without providing [`idHints`][idhints] then a
+new command would get registered, instead of overwriting the existing command. We therefore strongly recommend you to
+specify the [`idHints`][idhints].
+
+On top of the advantages in registering the command, having the ID of a command noted somewhere will also help you
+identify the command elsewhere in your code, such as in [interaction handlers][interaction-handlers].
+
+When Sapphire registers a application command it will log the command id for the command in console with the help of
+default logger provided by `Sapphire`. The command ID is logged on the `info` level, which is the default, however, if
+you have configured the Sapphire Logger to only log errors or fatal messages then please check
+[`Configuring Log Level Guide`][configure-logging] to learn how to change it.
+
+### `guildIds`
 
 This is an array of guild ids for which the command should be registered. By default it is `undefined`, which means that
 the command will be registered for all guilds (i.e. globally). If you want to register the command for a specific guild,
 you can pass in the guild id in the command registry.
 
-### registerCommandIfMissing
+:::note
+
+When the [`guildIds`][guildids] is an empty array the command will also be registered globally. This is useful for when
+you're getting the [`guildIds`][guildids] from an external source, i.e. through a `getGuildIds` function. You can make
+that external source default to an empty array which is often easier than `undefined`, especially when it comes from an
+external source that using a difference programming language from JavaScript/TypeScript (i.e. an external API or
+database).
 
-This is an optional that defaults to `true`. If set to `false`, the command will not be registered if it is not already
-registered i.e. if not already present in the `Discord API`.
+:::
 
-### behaviorWhenNotIdentical
+### `behaviorWhenNotIdentical`
 
 This options tells the registry that how to behave when the command is not identical to the one which is already
-registered on `Discord API`. By default it's set to [`LOG_TO_CONSOLE`][behavior-log] which means that it will warn you
-about the data differences found as compared to the data received from the `Discord API`.
+registered on Discord API. By default it's set to [`LOG_TO_CONSOLE`][behavior-log] which means that it will only warn
+you about the data differences found as compared to the data received from the Discord API.
 
-If you want the registry to `Overwrite` the command data, you can set this option to [`OVERWRITE`][behavior-overwrite].
+If you want the registry to be overwritten with your own local data, you can set this option to
+[`OVERWRITE`][behavior-overwrite]. Note that configuring [`OVERWRITE`][behavior-overwrite] will also still log the
```

Comment:
Half true, it logs the differences at debug level

Reply:
No, this is entirely true. Log levels are completely irrelevant to this sentence. And any log level is "will also still log...".

Also this is not really useful feedback. Either suggest a different wording, or don't say anything lol.

Resolving without change.